Add blog post on "How to Add MongoDB Compatibility to Neon" (#3863)
FerretDB/FerretDB: website/blog/2023-12-15-ferretdb-neon-add-mongodb-compatibility.md

### Open source

As an open source document database, you won't be at a risk of vendor lock-in.
Since MongoDB's license change to Server Side Public License (SSPL), there's [been a lot of disappointment and confusion with regards](https://blog.ferretdb.io/open-source-is-in-danger/) to how it affects many users and what it would mean for their applications.
According to the Open Source Initiative – the steward of open source and the set of rules that define open source software – SSPL is not considered open source.

In that regard, FerretDB, licensed under Apache 2.0, is a good option for users looking for a MongoDB alternative to return MongoDB workloads back to open source.

### Multiple backend options

At the moment, FerretDB supports Postgres and SQLite backends, with many ongoing works to support other backends.
Many databases built on Postgres can actually serve as a backend for FerretDB, including Neon.
That means you can take advantage of all the features available in that backend to scale and manage your database infrastructure without any fear of vendor lock-in.

## Prerequisites

- A Neon project and Postgres URI

> - FerretDB package

Reviewer: What is a FerretDB package? Maybe "Install FerretDB" or a link to the release pages would make this clearer.

- `mongosh`

## Create a Neon Project

Please refer to the instructions in the [Neon Sign up](https://neon.tech/docs/get-started-with-neon/signing-uphttps://neon.tech/docs/get-started-with-neon/signing-up) docs if this is your first project.

Before getting started with FerretDB, you need to create a Neon project.
When creating the project, you need to select the version of postgres, and also create a `ferretdb` database.

![Neon project creation](/img/blog/neon-create-project.png)

Once the project is created, Neon will provide a Postgres connection URI for you.
This URI will serve as the Postgres URI for your FerretDB instance.

## Install FerretDB on Linux

For this project, we will be using the FerretDB `arm` package, and you can access them from the [GitHub release page](https://github.com/FerretDB/FerretDB/releases).

> To do this, we'll need to curl the latest FerretDB package and then install it:
> 
> ```sh
> curl -LJO https://github.com/FerretDB/FerretDB/releases/download/v1.16.0/ferretdb-linux-arm64.deb
> ```
> 
> Then install FerretDB with:
> 
> ```sh
> sudo dpkg -i  ./ferretdb-linux-arm64.deb
> ```

Reviewer: I'd simplify this as

```suggestion
To do this, download and install the latest FerretDB release:

```sh
$ curl -LJO https://github.com/FerretDB/FerretDB/releases/download/v1.16.0/ferretdb-linux-arm64.deb
$ sudo dpkg -i  ./ferretdb-linux-arm64.deb
```
```

The "install" word twice sounds repetitive.
Also, do we have a link where if someone hits it, they are redirected to the latest available version? That'd be neat.

Author: Thanks @henvic. @AlekSi Is there a simplified way to get the latest version?

Reviewer 2: The simplest way to use the latest version is to use our Docker image. Is there any particular reason we use a Debian package with the cloud-native serverless database instead?
